tests/python/gaia-ui-tests/gaiatest/utils/email/email_util.py

```diff
@@ -28,8 +28,9 @@ def send(self, senders_account, mock_email):
                                        str(self.senders_account['smtp_port']))
```

Comment:
The changes in webqa-credentials will cause this to expect 'hostname' and 'port'. Please update the docstring and these occurrences.